src/pages/resultsView/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -5,7 +5,6 @@ import {computed, observable} from "mobx";
 import {observer} from "mobx-react";
 import Slider from 'react-rangeslider';
 import {FormGroup, ControlLabel, FormControl} from 'react-bootstrap';
-import jsPDF from 'jspdf';
```

Comment:
if you want, you could use the component I merged with my coexp PR, `DownloadControls.tsx`

Reply:
will do but after merge so it doesn't hold up review